blog/on_retros/README.md

```diff
@@ -0,0 +1,272 @@
+2) We started experimenting with recurring zoom "game nights." We did a few ad hoc starting at 5pm (so the day actually ended at 5) and if you wanted to you could join in to play code names, or among us, or jackbox games. Depending on how the team felt week to week, we would start to have these as often as every day.
+3) We gave everyone a personal action item to go ahead and schedule a bunch of PTO. In our experience, unlimited vacation days leads to people not taking vacations. So we tracked setting up vacation days in our sprints for a while.
+4) We also experimented with other mini end of day events like a "mini retro" every day when teams could demo their work for the day and get change management and UAT signoff and bring up issues we could discuss in standup the next morning.
+
+## Patterns!
+
+### Action Items Not Getting Done
+
+If action items aren't being done, the most common problems are that either they aren't tracked as part of normal sprint work, so they aren't top of mind for anyone, or the action items aren't concrete and actionable enough to actually get done.
+
+For the first case, as recommended early, I strongly suggest converting all action items to tickets at the top of the next sprint, so they are given the first class "normal day-to-day work" experience.
+
+For the latter case, I'd recommend applying the same standards you apply to your other engineering tickets to the action items from retros. That means clear context and well-defined acceptance criteria. Following the same principles as normal tickets in a sprint, the more well-defined the scope and AC, and the smaller the scope of the ticket, the easier it is to start and complete.
+
+### Tickets Rolling Over
+There are a lot of reasons velocity may be stuck and tickets are rolling over sprint to sprint. I'll cover some other bottlenecks below, but one of the first things to cover is to make sure your tickets are adequately groomed / refined and ready to go.
+
+If tickets are too big and poorly defined, it's hard to ever "finish" them. In general, the smaller the ticket, the mor explicit the scope, the more clear the problem context, the explicitness of the acceptance criteria, etc. all contribute to being able to actually execute and delivering some value. My recommendations to teams struggling with grooming tend to have two aspects
+1) What is a "Groomed Ticket"
+2) What is the process for grooming
+
+First, I'd recommend just getting the team to agree to some minimum requirement for a groomed ticket. For example, every Story needs 
+- User Stories: given in an `As <persona>, I want <X> so <reason>`. Again, this isn't supposed to replace building relationships with customers/stakeholders and having discussions, but having the context provided on each ticket from the perspective of the effected user is critical.
```

Comment:
I have love/hate with the user story pattern. I'd recommend this as a helpful recommendation rather than a definition.

When I was on Sushi I gave the team a few options that some found better to describe the problem in where the standard format was limited.

Not a big deal - just don't want this to come off wrong